enzyme/parsers/ebml/core.py

```diff
@@ -121,7 +121,7 @@ def get(self, name, default=None):
             raise ValueError('%s is a MasterElement' % name)
         return element.data
 
-    def getAll(self, name, default=[]):
+    def get_all(self, name, default=[]):
```

Comment:
Never use a list as default argument, you can read more on that extensively on the internet